service/templates/admin.html

```diff
@@ -630,6 +630,18 @@ <h2 class="modal-title">[[modalTitle]]</h2>
 										<label>Generate Task JSON Limit (MB)</label>
 										<input type="number" ng-model="Settings.task_limits.max_generate_task_json_size">
 									</md-input-container>
+									<md-input-container class="control" style="width:45%;">
+										<label>CPU Degraded Mode Parser Project Limit (MB)</label>
+										<input type="number" ng-model="Settings.task_limits.max_degraded_mode_parser_project_size">
+									</md-input-container>
+									<md-input-container class="control" style="width:45%;">
+										<label>Max Concurrent Large Parser Project Tasks</label>
+										<input type="number" ng-model="Settings.task_limits.max_concurrent_large_parser_project_tasks">
+									</md-input-container>
+									<md-input-container class="control" style="width:45%;">
+										<label>Max Parser Project Size</label>
+										<input type="number" ng-model="Settings.task_limits.max_parser_project_size">
+									</md-input-container>
```

Comment:
nit: might be slightly nicer if these were expressed in MB (or otherwise, if the admin settings page clarified that the unit is bytes vs. MB).